samples/Mvc.TagHelperSample.Web/Services/ISignalTokenProviderService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Primitives;
+
+namespace TagHelperSample.Web.Services
+{
+    public interface ISignalTokenProviderService<TKey>
```

Comment:
Doesn't need `TKey`. The implementation can assume the type. It's only ever used for `FeaturedMovies`. No need to make it more complex :smile: